templates/partials/bods-entity.html

```diff
@@ -23,8 +23,11 @@ <h5 class="card-header d-flex align-items-center">
             <span class="scorecard-value">{{ company.entity_name }}</span>
         </li>
         <li class="list-group-item {% if flags.company_id_flags.error %}list-group-item-danger{% elif flags.company_id_flags.warning %}list-group-item-warning{% endif %}">
-            <small class="scorecard-key">{{ company.identifiers_0_scheme }}</small>
-            <span class="scorecard-value">{{ company.identifiers_0_id }}</span>
+            <small class="scorecard-key">Company ID</small>
+            <span class="scorecard-value">
+                [{{ company.identifiers_0_scheme }}]
```

Comment:
@zarino These references have changed in #21 . May need to resolve.

Reply:
@SimKennedy Done!